<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>506</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QGroupBox" name="widget_groupBox">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>450</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>450</height>
      </size>
     </property>
     <property name="title">
      <string>AIM COMPONENT</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QFrame" name="widget_GroupFrame">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>420</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>420</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="1" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="name_label">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>0</height>
              </size>
             </property>
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Name:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="name_lineEdit">
             <property name="minimumSize">
              <size>
               <width>255</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QFrame" name="frame_4">
             <property name="minimumSize">
              <size>
               <width>75</width>
               <height>0</height>
              </size>
             </property>
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="9" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <widget class="QLabel" name="targetScale_label">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Target Scale:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="controlScale_doubleSpinBox">
             <property name="decimals">
              <number>3</number>
             </property>
             <property name="minimum">
              <double>-1000.000000000000000</double>
             </property>
             <property name="maximum">
              <double>1000.000000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.100000000000000</double>
             </property>
             <property name="value">
              <double>1.000000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="10" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_13">
           <item>
            <widget class="QLabel" name="controlStyle_label">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>0</height>
              </size>
             </property>
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Control Style:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="controlStyle_comboBox">
             <property name="currentIndex">
              <number>16</number>
             </property>
             <item>
              <property name="text">
               <string>box</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>pyramid</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>text</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>arrow</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>sphere</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>unit axes</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>arc arrow</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>arrows</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>sphere arrows</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>hand</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>circle</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>elipse</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>star</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>sun</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>gear</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>sparkle</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>eye</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>skull</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>bevel box</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Tippett Logo</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>lollipop</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>twist</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>arrow mid</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>arrow end</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item row="8" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="targetDistance_label">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Target Offset:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="controlOffset_doubleSpinBox">
             <property name="decimals">
              <number>3</number>
             </property>
             <property name="minimum">
              <double>-1000.000000000000000</double>
             </property>
             <property name="maximum">
              <double>1000.000000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.100000000000000</double>
             </property>
             <property name="value">
              <double>0.000000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="topJoint_label">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>0</height>
              </size>
             </property>
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Start Joint:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="startJoint_lineEdit">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="readOnly">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="startJoint_pushButton">
             <property name="text">
              <string>Load</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="3" column="0">
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="0" column="2">
            <widget class="QPushButton" name="endListAdd_pushButton">
             <property name="text">
              <string>Add</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QFrame" name="frame_6">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>50</height>
              </size>
             </property>
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Plain</enum>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="endList_label">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>End List:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1" rowspan="3">
            <widget class="QListWidget" name="endList_listWidget">
             <property name="maximumSize">
              <size>
               <width>370</width>
               <height>100</height>
              </size>
             </property>
             <property name="verticalScrollBarPolicy">
              <enum>Qt::ScrollBarAlwaysOn</enum>
             </property>
             <property name="horizontalScrollBarPolicy">
              <enum>Qt::ScrollBarAlwaysOff</enum>
             </property>
             <property name="sortingEnabled">
              <bool>true</bool>
             </property>
             <item>
              <property name="text">
               <string/>
              </property>
             </item>
             <item>
              <property name="text">
               <string/>
              </property>
             </item>
             <item>
              <property name="text">
               <string/>
              </property>
             </item>
             <item>
              <property name="text">
               <string/>
              </property>
             </item>
             <item>
              <property name="text">
               <string/>
              </property>
             </item>
             <item>
              <property name="text">
               <string/>
              </property>
             </item>
             <item>
              <property name="text">
               <string/>
              </property>
             </item>
             <item>
              <property name="text">
               <string/>
              </property>
             </item>
             <item>
              <property name="text">
               <string/>
              </property>
             </item>
             <item>
              <property name="text">
               <string/>
              </property>
             </item>
             <item>
              <property name="text">
               <string/>
              </property>
             </item>
             <item>
              <property name="text">
               <string/>
              </property>
             </item>
             <item>
              <property name="text">
               <string/>
              </property>
             </item>
             <item>
              <property name="text">
               <string/>
              </property>
             </item>
             <item>
              <property name="text">
               <string/>
              </property>
             </item>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QPushButton" name="endListRemove_pushButton">
             <property name="text">
              <string>Remove</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="4" column="0">
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <widget class="QTableWidget" name="aim_tabelWidget">
             <property name="minimumSize">
              <size>
               <width>300</width>
               <height>100</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>370</width>
               <height>100</height>
              </size>
             </property>
             <property name="verticalScrollBarPolicy">
              <enum>Qt::ScrollBarAlwaysOn</enum>
             </property>
             <property name="horizontalScrollBarPolicy">
              <enum>Qt::ScrollBarAlwaysOff</enum>
             </property>
             <property name="showGrid">
              <bool>false</bool>
             </property>
             <attribute name="horizontalHeaderDefaultSectionSize">
              <number>150</number>
             </attribute>
             <attribute name="horizontalHeaderMinimumSectionSize">
              <number>20</number>
             </attribute>
             <attribute name="verticalHeaderDefaultSectionSize">
              <number>28</number>
             </attribute>
             <column>
              <property name="text">
               <string>Aim joints</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Aim Sockets</string>
              </property>
             </column>
            </widget>
           </item>
           <item row="0" column="1" colspan="2">
            <widget class="QPushButton" name="aimAdd_pushButton">
             <property name="text">
              <string>Add</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QFrame" name="frame_2">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>50</height>
              </size>
             </property>
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Plain</enum>
             </property>
            </widget>
           </item>
           <item row="1" column="1" colspan="2">
            <widget class="QPushButton" name="aimRemove_pushButton">
             <property name="text">
              <string>Remove</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string notr="true"/>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QRadioButton" name="center_radioButton">
              <property name="text">
               <string>None</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="left_radioButton">
              <property name="text">
               <string>Lf</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="right_radioButton">
              <property name="text">
               <string>Rt</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
